jina/serve/instrumentation/__init__.py

```diff
@@ -0,0 +1,123 @@
+from typing import Any, Optional
+
+
+class InstrumentationMixin:
+    '''Instrumentation mixin for OpenTelemetery Tracing and Metrics handling'''
+
+    def _setup_instrumentation(
+        self,
+        name: str,
+        tracing: Optional[bool] = False,
+        span_exporter_host: Optional[str] = '0.0.0.0',
```

Comment:
I think for monitoring we have this naming the other way around, so `port_monitoring` and not `monitoring_port`. I think we should do the same and change this to `port_span_exporter` and `host_span_exporter`.